admin/app/views/commercial/highMerchandisingTargetedTags.scala.html

```diff
@@ -31,7 +31,9 @@
                         <td>@for(adUnitId <- lineItem.adUnitIds) {
                             <a href="@DfpLink.adUnit(adUnitId)">@{adUnitId}</a>
                         }</td>
-                        <td>@{lineItem.editionIfSet.distinct}</td>
+                        @if(lineItem.editionIfSet.isEmpty){
```

Comment:
If `editionIfSet` is just a list of editions against the line item then you can probably just rename it to `editions`; if it is `Nil` then it is fairly clear that there is no editional targeting on the line item. Likewise it is probably clear enough to leave a blank cell there when there is no targeting. Up to you!